PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+<!-- Delete Template sections if unneccesary -->
+<!-- Add issue number here (We encourage you to create the Issue First) -->
+<!-- You can also link the issue in Commit Messages -->
+
+Fixes #0000
+
+<!--
+Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!
+* [x] - Checked Box
+* [ ] - Unchecked Box
+-->
+
+### Tasks
+
+* [ ] Reviewed [contribution guidelines](https://github.com/nexB/scancode-toolkit/blob/develop/CONTRIBUTING.rst)
```

Comment:
Please fix the link to https://github.com/nexB/scancode-workbench/blob/develop/CONTRIBUTING.rst